packages/strapi-plugin-graphql/hooks/graphql/index.js

```diff
@@ -173,10 +181,22 @@ module.exports = strapi => {
       ) {
         serverParams.playground = {
           endpoint: strapi.plugins.graphql.config.endpoint,
+          subscriptionEndpoint: strapi.plugins.graphql.config.endpoint
         };
 
         serverParams.introspection = true;
       }
+      else if(['production', 'staging'].indexOf(process.env.NODE_ENV) >= 0 &&
+                process.env.REDIS_HOST){
+                  
+        serverParams.cache = new RedisCache({
+          host: process.env.REDIS_HOST,
```

Comment:
this is something that should probably be hooked into the database.json files instead of directly with an environment variable.

If possible it should use the redis package: https://github.com/strapi/strapi/tree/master/packages/strapi-hook-redis

As other plugins in the future (or even now) could also be using redis